apps/web/app/(app)/environments/[environmentId]/integrations/plain/actions.ts

```
@@ -0,0 +1,62 @@
+          type: "organization",
+          roles: ["owner", "manager"],
+        },
+        {
+          type: "projectTeam",
+          minPermission: "readWrite",
+          projectId,
+        },
+      ],
+    });
+
+    const encryptedAccessToken = symmetricEncrypt(key, ENCRYPTION_KEY!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)